frontend/RaffleBay/RaffleBay/View/Pre-Authorization Views/LoginView.swift

```diff
@@ -58,7 +66,13 @@ struct LoginView: View {
                         Button(action: {
                                 let response = login_request(email: self.email, password: self.password, authenticationVM: self.authenticationVM, user: self.newUser)
                                 //self.selection = response
+                                print(response)
+                            if(response == 1){
```

Comment:
I don't think this will work as it is. Alamofire is asynchronous, so the if(response == 1) could execute before line 67. You'll have to use completion handlers, which basically let you know when the request has finished executing. look at line 39, 62 in ProfileViewController. And then look at lines 119-123 in ProfileView.

Reply:
if you tried every case and it does work, then go ahead and merge, but just look into this to make sure